com.unity.render-pipelines.universal/Runtime/Passes/AdditionalLightsShadowCasterPass.cs

```diff
@@ -828,12 +828,12 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
         {
             if (m_CreateEmptyShadowmap)
             {
-                SetEmptyAdditionalShadowmapAtlas(ref context);
+                SetEmptyAdditionalShadowmapAtlas(ref context, renderingData.commandBuffer);
```

Comment:
should we pass RenderingData instead of cmd?

Reply:
that would also be consistent with RenderAdditionalShadowmapAtlas

Reply:
fixed in 3939959